src/test/resources/com/puppycrawl/tools/checkstyle/checks/descendanttoken/InputDescendantTokenTestLimitedToken.java

```diff
@@ -7,8 +7,6 @@
 */
 
 package com.puppycrawl.tools.checkstyle.checks.descendanttoken;
-
```

Comment:
Please restore empty line between package declaration and class declaration.

Reply:
did it sir !